### PR TITLE
README health check update

### PR DESCRIPTION
**Pull request by jgodson**
Update readme

### CODE REVIEW

**Review by lancelafontaine — APPROVED** [review]

File: README.md
Comment (slester) [inline README.md:21]:
```suggestion
- `oxygen_health_check`: Determines whether to ensure the preview URL is routable on Oxygen before marking the workflow as successful. A successful health check does **not** mean the application is accessible; it may be failing due to errors in the storefront's application code. The deployment status in the Hydrogen Storefronts channel will reflect the status of the deployment, not the health of the deployment. Accepts `true` or `false`. Defaults to `true`.
```

Comment (jgodson) [reply README.md:21]: Oh this popup up right after I hit the merge button 😢. I like the suggestion, will update 😄

Comment (jgodson) [reply README.md:21]: Though the problem I'm trying to highlight is more that a failed health check does not mean there is necessarily an issue with the deployment. The health check does account for some problems with the storefront application code (ie: a 500), but only on the root path.

**Pull request merged**
